### DISCUSSION TITLE
Resume Bundle X ATS Scoring

### DISCUSSION THREAD

**Discussion by yashksaini-coder** [recodehive/resume-pitch#99] [Q&A] [answered]
- Name: Yash Kumar Saini
- LinkedIn Link: [Linkedin](https://www.linkedin.com/in/yashksaini/)
- Resume Link: [Yash K. Saini Resume.pdf](https://github.com/user-attachments/files/17044703/Yash.K.Saini.Resume.pdf)
- GitHub Link: [Yash K. Saini](https://github.com/yashksaini-coder)
- Type of Job I am aiming for: Software Engineer roles, Data Scientist, ML Engineer 
- Small description about you: I am a final-year student currently pursuing BCA. I am passionate and interested in the field of Data Science. Have done few virtual (unpaid) internships in Data Science, Data Analysis & Machine Learning field.

**Comment by yashksaini-coder** [comment]
Hi @sanjay-kv just wanted to check if you got all the details and my resume. When will you provide an update ?

**Reply by sanjay-kv** [reply, marked as answer]
Looks better:
[yash.pdf](https://github.com/user-attachments/files/17089466/yash.pdf)
[ATS Version of CV (27).docx](https://github.com/user-attachments/files/17089467/ATS.Version.of.CV.27.docx)

@yashksaini-coder

**Reply by yashksaini-coder** [reply]
Thanks @sanjay-kv for resume review
